File: src/main/java/com/ysmull/easeshop/model/vo/ShopCartVO.java
package com.ysmull.easeshop.model.vo;

/**
 * @author maoyusu
 */
public class ShopCartVO {
    private Long userId;
    private Long goodsId;
    private String goodsName;
    private String picUrl;
    private String description;
    private Float price;
    private Integer amount;

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public Long getGoodsId() {
        return goodsId;
    }

    public void setGoodsId(Long goodsId) {
        this.goodsId = goodsId;
    }

    public String getGoodsName() {
        return goodsName;
    }

    public void setGoodsName(String goodsName) {
        this.goodsName = goodsName;
    }

    public String getPicUrl() {
        return picUrl;
    }

    public void setPicUrl(String picUrl) {
        this.picUrl = picUrl;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Float getPrice() {
        return price;
    }

    public void setPrice(Float price) {
        this.price = price;
    }

    public Integer getAmount() {
        return amount;
    }

    public void setAmount(Integer amount) {
        this.amount = amount;
    }

    @Override
    public String toString() {
        return "ShopCartVO{" +
                "userId=" + userId +
                ", goodsId=" + goodsId +
                ", goodsName='" + goodsName + '\'' +
                ", picUrl='" + picUrl + '\'' +
                ", description='" + description + '\'' +
                ", price=" + price +
                ", amount=" + amount +
                '}';
    }
}

File: EaseMall-biz/src/main/java/com/ysmull/easemall/model/vo/ShopCartVO.kt
package com.ysmull.easemall.model.vo

/**
 * @author maoyusu
 */
class ShopCartVO(
        var userId: Long,
        var goodsId: Long,
        var goodsName: String?,
        var price: Float,
        var amount: Int,
        var picUrl: String?,
        var description: String?) {


    override fun toString(): String {
        return "ShopCartVO{" +
                "userId=" + userId +
                ", goodsId=" + goodsId +
                ", goodsName='" + goodsName + '\''.toString() +
                ", picUrl='" + picUrl + '\''.toString() +
                ", description='" + description + '\''.toString() +
                ", price=" + price +
                ", amount=" + amount +
                '}'.toString()
    }
}
